### ISSUE TITLE
Can't import decks

### ISSUE THREAD

**Issue by LorduFreeman** [spacemandev-git/fvtt-card-support#79]
```
Uncaught (in promise) TypeError: Cannot read property 'name' of undefined
    at uploadFile (deck.js:562)
    at async deck.js:370
```
on importing any SDF deck zip - or trying to import images with a similar error. Tried with the 52 sample card deck as well.

Card Support 1.6.1
Foundry 0.8.6
DnD5 1.3.3

**Comment by LorduFreeman** [comment]
Correction: Importing as images works, but all images need to be present.  If for example the deck image is missing, the same error occurs. Maybe that is the core of this issue, SDF imports not providing images for the deck.

**Comment by Norc** [comment]
Yeah I'm not sure exactly what happened with this one, my test SDF import worked. I suspect something may have been off with the file somehow?

In the meantime, if anyone else has issues with SDF, try importing as images.

**Comment by StudentESE** [comment]
could you implement a refrash-button in the modules configuration so we can drag the files manually there. I think the only problem is the promize loading the zip for some reason. So maybe running a script only reads existing data to build up settings.db would help
